edx_solutions_api_integration/models.py

```diff
@@ -162,6 +163,12 @@ class Meta(object):
         proxy = True
 
 
+class LeaderBoard(TimeStampedModel):
+    user = models.ForeignKey(User)
```

Comment:
Add on_delete kwarg to ForeignKey

Context: 
Django 2.0 will make this field required for `ForeignKey` and `OneToOneFields`. In previous versions, this option defaulted to `models.CASCADE` when not specified. This change should make the deprecation warnings in the current Django version go away.